### ISSUE TITLE
Localization not working properly? [HELP NEEDED]

### ISSUE THREAD

**Issue by lauritzsh** [fulldecent/FDTake#78]
Hi, thanks for a neat library.

I have placed the `Localizable.strings` from the English and Danish under a `Supporting Files` group, and I can see they have been added to `en.lproj` and `da.lproj`. I can see they are picked up:

![image](https://cloud.githubusercontent.com/assets/1979410/12869536/035072e4-cd23-11e5-9c30-7614e678ba94.png)

The pop-up does not seem to be translated however, it will just say "takePhoto" for example. I know the `Localizable.strings` files are working, because I have this line

```swift
label.text = NSLocalizedString("loading", comment: "The loading message when pulling data")
```

and that changes correctly if I choose Danish or English.

Any idea if I am doing something wrong here? I copied them into the project, no reference link. I do not use CocoaPods. Content of the English `Localizable.strings`:

```swift
/* Decline to proceed with operation */
"cancel" = "Cancel";

/* Option to select photo from library */
"chooseFromLibrary" = "Choose from library";

/* Option to select photo from photo roll */
"chooseFromPhotoRoll" = "Choose from photo roll";

/* There are no sources available to select a photo */
"noSources" = "There are no sources available to select a photo";

/* Option to take photo using camera */
"takePhoto" = "Take photo";

/* Option to take video using camera */
"takeVideo" = "Take video";

/* The loading message when pulling data */
"loading" = "Loading...";
```

Thank you.

**Comment by fulldecent** [comment]
Could you please confirm first that the demo app is working for you in Danish?

**Comment by lauritzsh** [comment]
Running the app:

![image](https://cloud.githubusercontent.com/assets/1979410/12903101/6a22dfd6-cec5-11e5-98f1-75169675a8d5.png)

Please note "Operatør" in the top left, changed language to Danish inside the simulator (and scheme too just to be sure). It should have been "Vælg fra kamerarulle" instead of "chooseFromLibrary", right?

**Comment by fulldecent** [comment]
Thank you, I am still looking at this.

Does anyone else have advice on how to fix this?

**Comment by fulldecent** [comment]
I'm sorry for having the breaking bug in here so long.

The cocoapods updates are killing me and the normal approach of "`pod lib create FDTake` then start from scratch" each time did not work this time.

**Comment by fulldecent** [comment]
Was fixed and then broken again in Swift 3.0. Help requested, new issue at #100.
